.github/workflows/lint.yaml

```diff
@@ -0,0 +1,88 @@
+name: "Lint"
+
+on: [push, pull_request]
```

Comment:
```suggestion
on:
    push:
    pull_request:
```

Reply:
I like the compact notation more in this case.